static/js/todo_widget.js

```diff
@@ -73,9 +73,25 @@ export class TaskData {
                 // for legacy reasons, the inbound idx is
                 // called key in the event
                 const idx = data.key;
-                const key = idx + "," + sender_id;
                 const task = data.task;
                 const desc = data.desc;
+
+                if (!Number.isInteger(idx) || idx < 0 || idx > 99999) {
```

Comment:
This number seems pretty arbitrary. How'd you pick it?

Reply:
I changed the upper bound to be a `MAX_IDX` const with a comment explaining its value.